CODEOWNERS

```
@@ -5,16 +5,23 @@
 #
 # To be explicit: we will never accept changes to this file adding people from outside the Homebrew GitHub
 # organisation. If you are not a Homebrew maintainer: you do not personally "own" or "maintain" any formulae.
+#
+# Note: @Homebrew/plc does not have write-access to this repository, and therefore cannot be listed in this file.
+# To ensure PLC review for license-related PRs, @Homebrew/plc members who are also @Homebrew/core maintainers
+# should be listed individually for the following files:
+#   - CODEOWNERS
+#   - LICENSE.txt
+#   - audit_exceptions/permitted_formula_license_mismatches.json
 
-CODEOWNERS                        @Homebrew/tsc @Homebrew/plc @MikeMcQuaid 
+CODEOWNERS                        @Homebrew/tsc @MikeMcQuaid @issyl0 @SMillerDev
 CONTRIBUTING.md                   @Homebrew/tsc @MikeMcQuaid 
 tap_migrations.json               @Homebrew/tsc @MikeMcQuaid 
 .github/ISSUE_TEMPLATE/           @Homebrew/tsc @MikeMcQuaid 
 .github/PULL_REQUEST_TEMPLATE.md  @Homebrew/tsc @MikeMcQuaid 
-LICENSE.txt                       @Homebrew/plc @MikeMcQuaid
+LICENSE.txt                       @Homebrew/tsc @MikeMcQuaid @issyl0 @SMillerDev
```

Originally I just replaced `plc` with `tsc` here, but after adding Issy and Sean, should I remove the `tsc` requirement?

```suggestion
LICENSE.txt                       @MikeMcQuaid @issyl0 @SMillerDev
```